cluster.json

```diff
@@ -0,0 +1,58 @@
+[
+    {
+        "id": "brt",
+        "titel": "BRT",
+        "titel_kort":"BRT",
+        "beschrijving": "Beschrijvendetekst",
+        "beschrijving_kort": "Beschrijvendetekst",
```

Comment:
Achter laatste key/value paar hoort geen comma. Heb het gefixt in https://github.com/Geonovum/technisch-register/commit/9ef5c7b98dedd3a0d741c1d7b1c3d410e7adcd6b